metrics/metrics.go

```diff
@@ -47,6 +50,27 @@ func NewCommanderMetrics() *CommanderMetrics {
 	})
 	commanderMetrics.registry.MustRegister(commanderMetrics.MempoolSize)
 
+	commanderMetrics.MempoolSizeTransfer = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: namespace,
+		Name:      "mempool_size_transfer",
+		Help:      "number of pending transfers",
+	})
+	commanderMetrics.registry.MustRegister(commanderMetrics.MempoolSizeTransfer)
+
+	commanderMetrics.MempoolSizeCreate2Transfer = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: namespace,
+		Name:      "mempool_size_create2transfer",
+		Help:      "number of pending create2transfers",
+	})
+	commanderMetrics.registry.MustRegister(commanderMetrics.MempoolSizeCreate2Transfer)
+
+	commanderMetrics.MempoolSizeMassMigration = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: namespace,
+		Name:      "mempool_size_massmigration",
+		Help:      "number of pending massmigrations",
+	})
+	commanderMetrics.registry.MustRegister(commanderMetrics.MempoolSizeMassMigration)
+
```

Comment:
Instead of these four metrics use `Vec` to make this three tags on the one metric